### PR TITLE
[BitSail][Docs]Add FAQ document and footer to website.

### LINKED ISSUE #262: [BitSail][Docs] Add docs for computing parallelism.
## Is your feature request related to a problem? Please describe

There is no document about parallelism computing now.

## Describe the solution you'd like

Add a document to tell users how to set parallelism for BitSail connectors.

### PR DESCRIPTION
**Pull request by love-star**
Signed-off-by:

## Pre-Checklist

Note: Please complete **_ALL_** items in the following checklist.

- ✅ I have read through the [CONTRIBUTING.md](https://github.com/bytedance/bitsail/blob/master/docs/contributing.md) documentation.
- ✅ My code has the necessary comments and documentation (if needed).
- ✅ I have added relevant tests.

## Purpose
- Add FAQ document to website.
- Add FAQ footer to homepage.
- Close #262 

## Approaches

- Add FAQ document to website.
- Add FAQ footer to homepage.

N/A

### CODE REVIEW

File: website/.vuepress/footer.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ export const cnFooter = `
           <div style="text-align: center;">
             <h3>文档</h3>
             <a style="display: block; color: rgb(209, 213, 219); margin-bottom: 10px;" href="/bitsail/zh/documents/start/index.html">快速开始</a>
+            <a style="display: block; color: rgb(209, 213, 219); margin-bottom: 10px;" href="/bitsail/zh/documents/faq/README.md">FAQ</a>
```
Comment (BlockLiu) [inline website/.vuepress/footer.ts:14]: I think you should replace '.md' with '.html'.
And  you can test it locally (see https://bytedance.github.io/bitsail/zh/community/contribute.html#%E8%AF%B7%E6%B1%82%E4%BB%A3%E7%A0%81%E5%AE%A1%E6%9F%A5)

Comment (love-star) [reply website/.vuepress/footer.ts:14]: get it！

File: package.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
   "devDependencies": {
     "@vuepress/client": "2.0.0-beta.53",
     "vue": "^3.2.45",
-    "vuepress": "2.0.0-beta.53",
+    "vuepress": "^2.0.0-beta.59",
```
Comment (BlockLiu) [inline package.json:15]: Could you skip the modification to `package.json`  and `yarn.lock`, and add them in .gitignore file?

Comment (BlockLiu) [reply package.json:15]: These two files are not supposed to be changed.

Comment (love-star) [reply package.json:15]: OK!

**Pull request merged**